slices/grep.yaml

```diff
@@ -0,0 +1,20 @@
+package: grep
+
+slices:
+  bins:
+    essential:
+      - libc6_libs
+      - libpcre3_libs
+    contents:
+      /bin/grep:
+
+  deprecated:
+    # These are shell scripts requiring a symlink from /usr/bin/bash to /bin/sh
+    # See: https://manpages.ubuntu.com/manpages/jammy/en/man1/grep.1.html
+    essential:
+      - bash_bins
```

Comment:
I am wondering if we need `dash` or `bash`. If you only need classic `/bin/sh`, `dash` seems to be the better choice.